bootstrap/bootstrap_reviews.py

```diff
@@ -0,0 +1,174 @@
+import csv
+import requests
+import datetime, json, os
+
+
+# A JWT generated for a service Moderator user
+AUTH = os.getenv("MODERATOR_AUTH")
+
+# The URL of the Moderator API
+API_URL = os.getenv("MODERATOR_API", "127.0.0.1:8080")
+
+# The number of reviews per category to import into Moderator
+REVIEWS_PER_CATEGORY = 100
```

Comment:
Rename variable? I think better to use moderator terminology here, i.e.e COMMENTS_PER_ARTICLE?